static/openapi/p3-spec.json

```diff
@@ -0,0 +1,3479 @@
+            "200": {
+              "description": "Usage information retrieved.",
+              "headers": {
+                "X-count": {
+                  "description": "The total number of records for the specified time period.",
+                  "schema": {
+                    "type": "number",
+                    "default": 1000
+                  }
+                }
+              },
+              "content": {
+                "application/json": {
+                  "schema": {
+                    "oneOf": [
+                      {
+                        "type": "array",
+                        "items": {
+                          "$ref": "#/components/schemas/UsageGetResponse"
+                        }
+                      },
+                      {
+                        "type": "array",
+                        "items": {
+                          "$ref": "#/components/schemas/AggregateUsageGetResponse"
+                        }
+                      }
+                    ]
+                  },
+                  "examples": {
+                    "example": {
+                      "summary": "Response when aggregate is false.",
+                      "value": [
+                        {
+                          "serverUrl": "something.com",
+                          "requests": 1305,
+                          "maxRps": 100,
+                          "timestamp": "2023-06-12T00:00:00.000Z"
+                        },
+                        {
+                          "serverUrl": "something.com",
+                          "requests": 1200,
+                          "maxRps": 125,
+                          "timestamp": "2023-07-12T00:00:00.000Z"
+                        }
+                      ]
+                    },
+                    "example_two": {
+                      "summary": "Response when aggregate is true.",
+                      "value": [
+                        {
+                          "serverUrl": "something.com",
+                          "requests": 2505,
+                          "maxRps": 112.5
+                        }
+                      ]
+                    }
+                  }
+                }
+              }
+            },
+            "401": {
+              "description": "Authorization failure due to invalid authentication credentials."
+            },
+            "403": {
+              "description": "Either the API key doesn't have permissions or it is deactivated."
+            },
+            "404": {
+              "description": "Usage info not available."
+            },
+            "500": {
+              "description": "Internal server error."
+            }
+          }
+        }
+      },
+      "/api/p3/v1/cache/purge": {
+        "post": {
+          "tags": [
+            "Cache"
+          ],
+          "summary": "Purge cache",
+          "description": "Purges URLs from the cache. Can also purge CDN cache if specified.",
```

Comment:
```suggestion
          "description": "Purge URLs from the cache. Can also purge CDN cache if specified.",
```

Reply:
I wonder if we need to specify what cache?  The "P3 cache"?  When I read this the first time I did not know which cache you were talking about.